src/main/plugin/iso19139.rndt/layout/config-editor.xml

```diff
@@ -566,8 +566,23 @@
 
           <field name="datasetcode"
                  xpath="/gmd:MD_Metadata/gmd:identificationInfo/*
-                 /gmd:citation/gmd:CI_Citation/gmd:identifier/gmd:RS_Identifier/gmd:code"
-                 or="identifier" in = "/gmd:MD_Metadata/gmd:identificationInfo/*/gmd:citation/gmd:CI_Citation" />
+                 /gmd:citation/gmd:CI_Citation/gmd:identifier/*/gmd:code"/>
+
+          <action type="add" btnLabel="datasetcode" or="identifier" in="/gmd:MD_Metadata/gmd:identificationInfo/*/gmd:citation/gmd:CI_Citation"
+                  if="count(gmd:MD_Metadata/gmd:identificationInfo/*/gmd:citation/gmd:CI_Citation/gmd:identifier)=0"
+          >
+            <template>
+              <snippet>
+                <gmd:identifier>
+                  <gmd:MD_Identifier>
+                    <gmd:code>
+                      <gco:CharacterString></gco:CharacterString>
+                    </gmd:code>
+                  </gmd:MD_Identifier>
+                </gmd:identifier>
+              </snippet>
+            </template>
+          </action>
```

Comment:
Perchè questo bottone?
Quel campo dovrebbe sempre essere valorizzato in automatico dall'UFI.